packages/components/CHANGELOG.md

```diff
@@ -4,10 +4,13 @@
 
 ### Enhancements
 
+-   Export `IconType` type for external use. ([#49649](https://github.com/WordPress/gutenberg/pull/49649))
+
+### Enhancements
```

Comment:
We'll need to move this accordingly since the Enhancements section already exists.